docs/src/main/asciidoc/getting-started.adoc

```diff
@@ -818,7 +818,7 @@ to the following in the build logs:
 [[getting-started-cdc]]
 == Step-by-step Guide to Consumer Driven Contracts (CDC) with Contracts on the Producer Side
 
-Consider an example of fraud detection and the loan issuance process. The business
+Consider an example of fraud detection, and the loan issuance process. The business
```

Comment:
Do not add this comma. Again, this conjunction ("and") is creating a two-item list rather than attaching an independent clause.